docs/src/content/docs/getting-started/setup.md

```diff
@@ -5,107 +5,35 @@ sidebar:
   order: 1
 ---
 
-Reatom is a framework-agnostic state manager, and you can use it with various adapters for different frameworks. This guide provides a common usage with React.js, as it is the most commonly used view library currently.
```

Comment:
why is this This should be left out, without this mention the examples on reatom give the false idea that reatom is only compatible with reatom